src/index.js

```diff
@@ -202,30 +218,36 @@ const Api = (options = {}) => {
         ...hdl
       ]
         .reduce((p, c) => p.then(c), Promise.resolve())
-        .then(async (payload) => generateResponse(null, payload, rb, {
+        .then(async (payload) => generateResponse(null, payload, {
           defaultHeaders: await generateDefaultHeaders(event.headers)
         }))
-        .catch(async (err) => generateResponse(err, null, rb, {
+        .catch(async (err) => generateResponse(err, null, {
           defaultHeaders: await generateDefaultHeaders(event.headers)
         }));
+      const statusCode = get(result, 'statusCode');
```

Comment:
```suggestion
      const statusCode = result.statusCode;
```

Reply:
Right, because we can be sure that the statusCode is present on result